README.md

```diff
@@ -4,18 +4,18 @@ User interface for MetalKube
 
 ## Getting started
 
-* Install [yarn][1] and [golang][2]
```

Comment:
Why do you hate stars? :smile: :star:

Reply:
Prettier seems to hate them. /me updates vscode config.